### PR TITLE
Having very very low sanity will rarely cause hallucintions

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

- Being on sanity level "Crazy" and "Insane" will rarely cause hallucinations
   - Crazy: every ~4-8 minutes
   - Insane: every ~2-4 minutes
   - Using Psicodine will stop this
- Renamed `TRAIT_HALLUCINATION_SUPPRESSED` to be called `TRAIT_RDS_SUPPRESSED`, as it's a bit less misleading
- Removing RDS from admin VV will actually remove the associated effects / trauma

## Why It's Good For The Game

- People rarely interact with the mood system, and having more incentives to would be neat. 
   - With the exception of moving like a snail, being insane is barely noticeable. Sure, it affects actionspeed and crit threshold, but these are very hidden effects.
- A lot of people never see hallucinations, which I just find a bit of a shame, as there's a few new ones people will never see. 
   - The only time like 95% of jobs see a hallucination is a mass hallucination.
   - Not counting for RDS, pretty much the only job liable to get hallucinations is engineering.
- The main sources of hallucinations on the station come at you fast, which makes it easy to exhaust all the possible ones and takes away their charm.
 
So, we can tackle all of these in a pretty sensible way: By having low sanity (on rare occasions) cause hallucinations.

This sprinkles in a few hallucinations to sow doubt, while keeping them from being overbearing. All the while, also nudging the player to get their mind in a better shape. I thought it sounded pretty neat.

## Changelog

:cl: Melbert
add: Having very low sanity (insane or crazy, the last two sanity levels) will cause you to experience hallucinations rarely.
admin: Removing RDS via the VV menu will actually remove the associated effects.
/:cl:

### CODE REVIEW

**Comment by calliophis-maculiceps** [comment]
it would be better if the slowdown was toned down and low sanity had even more hallucinations than what you propose, as rp wise its more interesting to have someone have a few hallucinations than simply them being basically crippled

**Comment by GrimsonGrime** [comment]
Awwww your doing my big project idea for me 😢

File: code/__DEFINES/traits.dm [hunk 1]
```diff
@@ -318,7 +318,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// prevents the damage done by a brain tumor
 #define TRAIT_TUMOR_SUPPRESSED "brain_tumor_suppressed"
 /// Prevents hallucinations from the hallucination brain trauma (RDS)
-#define TRAIT_HALLUCINATION_SUPPRESSED "hallucination_suppressed"
+#define TRAIT_RDS_SUPPRESSED "hallucination_suppressed"
```
Comment (JohnFulpWillard) [inline code/__DEFINES/traits.dm:321]:
```suggestion
#define TRAIT_RDS_SUPPRESSED "rds_suppressed"
```

**Comment by Bluedino1025** [comment]
How would this affect people who already have RDS?

**Comment by thebestcodertoevercode** [comment]
Amnesia pog

**Pull request merged**